content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+ ---
+reviewers:
+- caesarxuchao
+- dchen1107
+title: नोड्स
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Kubernetes _Nodes_ पर चलने के लिए पॉड्स में कंटेनर रखकर आपका {{< glosary_tooltip text="workload" term_id="workload" >}} चलाता है।
+क्लस्टर के आधार पर एक नोड वर्चुअल या भौतिक मशीन हो सकता है। प्रत्येक नोड
+द्वारा प्रबंधित किया जाता है
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}
+और चलाने के लिए आवश्यक सेवाएं शामिल हैं
+{{< glosary_tooltip text="Pods" term_id="pod" >}}.
+
+आम तौर पर आपके पास क्लस्टर में कई नोड होते हैं; एक सीखने या संसाधन-सीमित में
+पर्यावरण, आपके पास केवल एक नोड हो सकता है।
+
+नोड पर [घटक](/docs/concepts/overview/components/#node-components) में शामिल हैं
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}},
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}, और
+{{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}.
+
+<!-- body -->
+
+## प्रबंध
+
+{{< glosary_tooltip text="API server" term_id="kube-apiserver" >}} में नोड्स जोड़ने के दो मुख्य तरीके हैं:
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
{{< glossary_tooltip text="API सर्वर" term_id="kube-apiserver" >}} में नोड्स जोड़ने के दो मुख्य तरीके हैं:
```